.github/workflows/automatic-updates.yml

```diff
@@ -32,5 +32,3 @@ jobs:
           commit-message: "feat: Node.js ${{ steps.updt.outputs.result }}"
           title: "feat: Node.js ${{ steps.updt.outputs.result }}"
           delete-branch: true
-          team-reviewers: |
-            @nodejs/docker
```

Comment:
I'm wondering if the issue is actually the leading `@`. The docs are a little sparse on the property

Reply:
Ooh. That might be! Not sure how to test it... Ideas?

Reply:
We could try dropping it and seeing if the message is still there on the next PR I guess. I think the message only happens when the actual PR creation is run, rather than on the regular cron jobs with no changes

Reply:
I'm down with that. Send a PR doing so? Hopefully we remember to verify 😅